### PR TITLE
chore: add more repos

### PR DESCRIPTION
**Pull request by marten-seemann**
According to the most recent run of the repo health script (see #43), CI will succeed in these repos.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -130,12 +135,15 @@
...
   { "target": "multiformats/go-multiaddr" },
   { "target": "multiformats/go-multiaddr-dns" },
   { "target": "multiformats/go-multiaddr-fmt" },
   { "target": "multiformats/go-multibase" },
   { "target": "multiformats/go-multicodec" },
   { "target": "multiformats/go-multihash" },
   { "target": "multiformats/go-multistream" },
-  { "target": "multiformats/go-varint" }
+  { "target": "multiformats/go-varint" },
+  { "target": "multiformats/ma-pipe" },
```
Comment (mvdan) [inline config.json:148]: hmm, trailing newline - should we make CI on PRs fail if the json syntax is invalid?

**Review by mvdan — APPROVED** [review]

File: config.json (hunk 1 above)
Comment (mvdan) [reply config.json:148]: https://github.com/protocol/.github/issues/128 would probably have caught this, as it needs to parse the json anyway.

Comment (marten-seemann) [reply config.json:148]: I'm surprised `jq` didn't catch it. It's run as part of check_config.sh.

Comment (marten-seemann) [reply config.json:148]: It's especially weird since locally jq fails: `parse error: Expected another array element at line 149, column 1`.

Comment (marten-seemann) [reply config.json:148]: Actually, it did throw an error, but for some reason not fail the build: 
image

Comment (marten-seemann) [reply config.json:149]: Rebased on top of #149, and CI is now actually failing.

**Review by Stebalien — APPROVED** [review]

File: config.json [hunk 2]
```diff
@@ -26,6 +28,7 @@
   { "target": "ipfs/go-fetcher" },
   { "target": "ipfs/go-filestore" },
   { "target": "ipfs/go-fs-lock" },
+  { "target": "ipfs/go-hamt-ipld" },
```
Comment (Stebalien) [inline config.json:31]: Hm. Actually, I'm going to archive this repo.

Comment (marten-seemann) [reply config.json:31]: Great. A test for our cron workflow!

Comment (Stebalien) [reply config.json:31]: Ok, closing https://github.com/protocol/.github/pull/151.

**Pull request merged**